src/utils/CAPTCHADetector.ts

```
@@ -0,0 +1,1163 @@
+                )
+            }
+
+            // Check for visual CAPTCHA elements
+            let captchaFound = false
+            let captchaType: 'text' | 'image' | 'audio' = 'text'
+            let captchaSelector = ''
+
+            // Check for text-based CAPTCHA input fields
+            for (const selector of Object.values(
+                this.CAPTCHA_SELECTORS.input,
+            )) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)